config/site_settings.yml

```diff
@@ -2578,8 +2578,13 @@ tags:
   tags_listed_by_group:
     client: true
     default: false
-  allow_staff_to_tag_pms:
```

Comment:
Just a note but if this site setting is no longer used, we need a migration to drop the row from the database if it exists.